src/Storages/MergeTree/MergeTreeIndexSet.cpp

```diff
@@ -554,7 +554,7 @@ void MergeTreeIndexConditionSet::traverseAST(ASTPtr & node) const
     if (atomFromAST(node))
     {
         if (node->as<ASTIdentifier>() || node->as<ASTFunction>())
-            node = makeASTFunction("__bitWrapperFunc", node);
+            node = makeASTFunction("__bitWrapperFunc", makeASTFunction("toUInt8OrDefault", node));
```

Comment:
Missing comment.

I don't understand this code.
1. Why did it work before? The bitWrapperFunc should throw an exception for incorrect types.
2. How should it work for String fields? Is returning 0 as default correct?

Reply:
This is an attempt to mute the default implementation for Null. Maybe using `ifNull` is better, but it has 2 args.
Will make it more explicit.